apps/browser/qml/pages/SettingsPage.qml

```diff
@@ -166,4 +170,15 @@ Page {
         key: "/apps/sailfish-browser/settings/home_page"
         defaultValue: "http://jolla.com/"
     }
+
+    DataFetcher {
```

Comment:
I'd prefer seeing "install" invokable on the searchenginemodel which would be using datafetcher.

As after installing it, you need to update the datamodel as well.